src/components/plots/TrajectoryAnalysisPlot.jsx

```diff
@@ -88,10 +85,6 @@ const TrajectoryAnalysisPlot = forwardRef((props, ref) => {
     }
   }, [experimentId, embeddingMethod, embeddingSettings]);
 
-  useEffect(() => {
-    changeEmbeddingAxesIfNecessary(config, embeddingMethod, onUpdate);
-  }, [config, embeddingMethod]);
-
```

Comment:
Great to see that this can be removed. With this code deleted, how do the axes get changed now? Where in the code is this handled now?

Reply:
with this removed, the embedding axes titles are handled in the specs themselves:

if the axis does not have a name its set to a default one `embeddingMethod-0/1` so `UMAP/TSNE-0/1` 0 if its x axis 1 if its y axis